pkg/api/hosttwin.go

```diff
@@ -0,0 +1,62 @@
+package api
+
+import (
+	"fmt"
+
+	"github.com/Iotic-Labs/iotics-identity-go/pkg/advancedapi"
+	"github.com/Iotic-Labs/iotics-identity-go/pkg/register"
+)
+
+// DelegateControlByPrivateExponent registers a twin identity with a control delegate to agent
+func DelegateControlByPrivateExponent(resolverClient register.ResolverClient, twinIssuer *register.Issuer, twinPrivateExponent string, agentID register.RegisteredIdentity, delegationName string) error {
```

Comment:
Worth having *hex* either in the the input var or func name? (then would be in line with other funcs like `GetKeyPairFromPrivateExponentHex`) .. or would that be too wordy?